driver/fuzz_target_runner.cpp

```diff
@@ -197,19 +197,20 @@ RunResult FuzzTargetRunner::Run(const uint8_t *data, const std::size_t size) {
     env.DeleteLocalRef(byte_array);
   }
 
-  if (env.ExceptionOccurred()) {
-    jlong dedup_token = computeDedupToken();
+  if (jthrowable exception = env.ExceptionOccurred()) {
+    env.ExceptionClear();
```

Comment:
Is it possible that `ExceptionClear` frees the memory of the exception object? That might explain the crash on macOS.

Reply:
That's a good idea, I have pushed a speculative fix and will see what the pipeline says. Unfortunately the JNI doc is completely unclear on this.

Reply:
This segfault does not seem to have anything to do with this PR. Large Java stacks seems to have a history of issues on macOS. I'll mark the test as broken.